examples/graphql-helix-defer-stream/package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "dependencies": {
     "fastify": "3.29.0",
-    "@envelop/core": "2.3.3",
+    "@envelop/core": "*",
```

Comment:
Why only here a star (`*`)?

Reply:
all examples are `*` so making this consistent